app/src/ui/dispatcher/error-handlers.ts

```diff
@@ -510,7 +510,10 @@ export async function refusedWorkflowUpdate(
 
   const rejectedPath = match[1]
 
-  if (!rejectedPath.startsWith('.github/workflows/')) {
+  if (
+    rejectedPath.startsWith('.github/') &&
+    rejectedPath.indexOf('workflow') >= 0
```

Comment:
❓ Also, I'm curious about context on making this check more lax in 1e0f385881e589931c1336d6c5e78a7b1ea0f52a, as my understanding was that having `.github/workflows` as a folder was [required](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file)

Reply:
HCI syntax workflow files were only deprecated a few days ago (after I wrote this PR even) and the logic on the dotcom side still includes these older path-styles so by being lax we match all paths that dotcom might reject.